gensim/models/word2vec.py

```diff
@@ -1116,17 +1116,33 @@ def doesnt_match(self, words):
         dists = dot(vectors, mean)
         return sorted(zip(dists, words))[0][1]
 
-    def __getitem__(self, word):
+    def __getitem__(self, words):
+
         """
-        Return a word's representations in vector space, as a 1D numpy array.
+        Accepts a single word (or) a list of words as input. 
+
+        Returns a word's representations in vector space, as a 1 X VEC_LENGTH 1D numpy array, IF a word is passed as an argument (or)
+        Returns N word's representations in vector space, as a N X VEC_LENGTH 2D numpy array, IF a list of words are passed as an argument.
```

Comment:
Minor doc comment tweaks: should hard-wrap for easier reading; possessive plural `N words'`; example below (L1134) should have space after comma.